### PR TITLE
TODO Try 2:  clean up some old comments, remove screenToShowOnPause, …

### PR DESCRIPTION
**Pull request by lental**
…enforce required sections of the json

Going to get some of this stuff out of our repo.  please check it out and provide feedback

**Breaking Changes**: If you use screenToShowOnPause (which is not used on Native), then you should be fine.

validate works fine

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -63,8 +63,7 @@
     },
     "showTitle": true,
     "showDescription": true,
-    "infoPanelPosition": "topLeft",
-    "screenToShowOnPause": "default"
```
Comment (ksunika) [inline skin.json]: I think we are using screenToShowOnPause in our onPaused() function.

**Comment by lental** [comment]
reverted :)

**Comment by lental** [comment]
bump - can this be merged?

**Comment by ksunika** [comment]
:+1:

**Comment by lental** [comment]
don't forget to delete branches after the merge!

**Pull request merged**